app/table/cells/DefaultPoolCell.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+import { Success12Icon } from '@oxide/design-system/icons/react'
+
+import { Badge } from '~/ui/lib/Badge'
+
+import type { Cell } from './Cell'
+
+export const DefaultPoolCell = ({ value }: Cell<boolean>) =>
```

Comment:
Nitpick, but I think this `Cell` type is pointless. I want to get rid of it in a followup. Might as well not use it here.

```suggestion
export const DefaultPoolCell = ({ value }: { value: boolean }) =>
```

And a nitpick on the nitpick, since we're no longer constrained by `QueryTable` to have the prop named `value`, it might be nice to name it something more descriptive like `isDefault`.